projects/ffmpeg/build.sh

```diff
@@ -92,24 +77,17 @@ then
       export CC=clang
       export CXX=clang++
       export FFMPEG_CFLAGS="-g -fprofile-instr-generate -fcoverage-mapping"
-      export LDFLAGS="$FFMPEG_CFLAGS"
-      export LINK_FLAGS="$LDFLAGS -ldrm -lm -ldl -lXext -lz -lpthread -lrt -L/ffmpeg_deps/lib \
-                        -L/ffmpeg_deps/lib/alsa-lib/smixer/ -L/ffmpeg_deps/lib/vdpau/ \
-                        -I/ffmpeg_deps \
-                        -lfdk-aac -lvorbisenc -lvorbisfile -l:smixer-hda.a -l:smixer-ac97.a \
-                        -l:smixer-sbase.a -lvdpau_trace -lavfilter -lavdevice -lpostproc \
-                        -lva -logg -lswscale -ltheoradec -ltheoraenc -lvpx -lasound -lbz2 \
-                        -lswresample -lvorbis -lvdpau -lavutil -lavcodec -lavformat \
-                        -ltheora -lva-drm -lxml2 -lopus -fuse-ld=/usr/bin/ld.lld"
+      export CFLAGS="$FFMPEG_CFLAGS"
+      export LINK_FLAGS="$FFMPEG_CFLAGS $LINK_LIBS"
       export TARGET_BSF="-o /target_bsf_cov target_bsf_fuzzer.o main.o"
       export TARGET_DEC="-lxcb -lxcb-shm -lxcb -lxcb-xfixes -lxcb -lxcb-shape -lxcb -lX11 -lbz2 \
                          -o /target_dec_cov target_dec_fuzzer.o main.o"
       export TARGET_DEM="-o /target_dem_cov target_dem_fuzzer.o main.o"
 fi
 
 # Disable UBSan vptr since several targets built with -fno-rtti.
-export CFLAGS="$LDFLAGS -fno-sanitize=vptr"
-export CXXFLAGS="$LDFLAGS -fno-sanitize=vptr"
+export CFLAGS="$FFMPEG_FLAGS"
```

Comment:
typo: `FFMPEG_CFLAGS`  
Add `-fno-sanitize=vptr` to CFLAGS either

Reply:
@PaDarochek 
`-fno-sanitize=vptr` is not needed for C code. Most of the `ffmpeg` project is written on C, so this is the way to build with `UBSAN`, by passing `-fsanitize=undefined` option from `FFMPEG_FLAGS` to `CFLAGS`.
This is ok.

Reply:
Ok, so just fix typo

Reply:
fixed